frontend/courses/sample/tour/math.evy

```diff
@@ -0,0 +1,14 @@
+pi := 3.141593
```

Comment:
FYI: through experimentation I found that these values:
```
pi := 3.14159265
e := 2.718281828459045
```
Make `log e` print `1` and `cos pi` print `-1`. `sin pi` is still not quite zero though, even when I add a bunch more precision.